recipe/meta.yaml

```diff
@@ -49,7 +49,7 @@ about:
   license_family: BSD
```

Comment:
License is BSD-2-Clause, see https://github.com/nose-devs/nose2/blob/f65060dee270ccaa5e3e9561524c0335001fcfca/pyproject.toml#L16 but the license relies on unittest's BSD-3-Clase too https://github.com/nose-devs/nose2/blob/0.14.0/LICENSE

I would say the license is `BSD-2-Clause AND BSD-3-Clause`

Reply:
done